.travis.yml

```diff
@@ -17,30 +17,59 @@ env:
     #   See https://docs.travis-ci.com/api#external-apis.
     - secure: "qQ3vMSXUybUyKljJjgcVT9YaWcZ0PkdLZSFj6BVuo82SWO7e4L+hC8pqsa1Sm/EjyetVwfKk5lkvruPpro++8hDPLq3ahNOMmm4axfcWvACpW+acD/VCA5D8Dxz2ohrvpwsqo7BmIV6OUtIO5JeePHP1SqaIdVZFJ0pIAZToEWQ="
     - CONDA_ROOT_PYTHON_VERSION: "2.7"
-    - NUMPY_VERSION=1.11.1
-    - PANDAS_VERSION=0.18.1
-    - SCIPY_VERSION=0.17.1
+  matrix:
+    - PANDAS_18=1
```

Comment:
What do you think about calling this `OLD_PANDAS` and `NEW_PANDAS` instead of encoding versions? I think the long term model we'd like to have is to support a "stable" version of pandas for use on Quantopian, and a relatively new release for general consumption.

Reply:
I had this internal debate as well. I don't have a super strong opinion, but the two thoughts I had before going with the proposed vars were:

1. When I go to the travis build matrix, I actually find it useful to know what versions are being tested, without having to click through to see that NEW=0.22.
1. Will there be transition periods where we support three versions briefly, or do you think the cut-overs will cleanly update the NEW version? If we ever have three, then again, I think explicit versions will be clearer than OLD, NEW, NEWER.

Reply:
> Will there be transition periods where we support three versions briefly, or do you think the cut-overs will cleanly update the NEW version?

I don't think it's going to be feasible for us to realistically support more than two build configurations given the resources we currently have. I think the most likely model for transitions will be that we'll either update `OLD_PANDAS` to match `NEW_PANDAS`, in which case they'll be the same for a bit (and maybe we comment out the `NEW_PANDAS` builds temporarily) or we update `OLD_PANDAS` to some newer version that's still not as new as the newest (e.g., if we want to bump the version on Quantopian but can't upgrade all the way to the latest for some reason).

> When I go to the travis build matrix, I actually find it useful to know what versions are being tested, without having to click through to see that NEW=0.22.

I feel like we update pandas versions infrequently enough that it's not too hard to keep track of the OLD/NEW versions.

Overall, my instinct is that making it easier to just increase the `NEW_PANDAS` version will make it more likely that we actually keep the new version up to date, and that seems more valuable to me than the points you raised. I don't have a super strong opinion about this though.

Reply:
Sounds good, and we can always change it later.